.ci/scripts/doca_install.sh

```diff
@@ -58,6 +58,10 @@ function map_os_and_arch {
 
         rhel|ol)
             OS="${ID}${VERSION_ID}"
+            if [[ "$VERSION_ID" =~ "9.4" ]]; then
```

Comment:
can you add comment that explain why we need special version here and that it should be changed in the future.

Reply:
Added the explanation in comment